tests/data/SD_ME0WME0W/dataset_ingest_config.yml

```diff
@@ -0,0 +1,46 @@
+
+### Dataset Ingest Config ###
+
+# The list of entities that will be loaded into the target service
+target_service_entities:
+    - 'investigator'
+    - 'study'
+    - 'study_file'
+    - 'family'
+    - 'participant'
+    - 'family_relationship'
+    - 'diagnosis'
+    - 'phenotype'
+    - 'outcome'
+    - 'biospecimen'
+    - read_group
```

Comment:
Why are some of these wrapped in ' and some not?

Reply:
Idk, I copied and pasted from test_study. I can fix 🛠